api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticSearchService.scala

```diff
@@ -0,0 +1,38 @@
+package uk.ac.wellcome.platform.api.services
+
+import javax.inject.{Inject, Singleton}
+
+import com.sksamuel.elastic4s.ElasticClient
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.twitter.inject.annotations.Flag
+import uk.ac.wellcome.platform.api.models._
+import uk.ac.wellcome.utils.GlobalExecutionContext.context
+
+import scala.concurrent.Future
+
+@Singleton
+class ElasticSearchService @Inject()(@Flag("es.index") index: String,
+                                     @Flag("es.type") itemType: String,
+                                     elasticClient: ElasticClient) {
+
+  def findRecordById(canonicalId: String): Future[Option[Record]] =
+    elasticClient
+      .execute {
+        search(s"$index/$itemType").query(
+          boolQuery().must(matchQuery("canonicalId.keyword", canonicalId))
+        )
+      }
+      .map { _.hits.headOption.map { Record(_) } }
+
+  def findRecords(): Future[Array[Record]] =
+    elasticClient
+      .execute {
+        search(s"$index/$itemType")
+          .matchAll()
+          // Sort so that we always have a consistent result that we can assert on
+          .sortBy(fieldSort("canonicalId.keyword"))
+          .limit(10)
+      }
+      .map { _.hits.map { Record(_) } }
```

Comment:
Absolutely right. Default order TBC, but likely to be based on one of the date fields.